src/thunderbots/software/grsim_communication/motion_controller.cpp

```diff
@@ -17,13 +17,20 @@
 
 #include "util/logger/init.h"
 
+template <class... Ts>
```

Comment:
A comment indicating why this is here (so we can use the `overloaded` thing with the `std::visit` below), and maybe link to https://en.cppreference.com/w/cpp/utility/variant/visit would be very helpful.

This even threw me off when I first read it before I scrolled down and realized what it was for.